server/package.json

```diff
@@ -7,7 +7,6 @@
   "scripts": {
     "dev": "nodemon | npx pino-pretty",
     "start": "node server",
-    "test": "NODE_OPTIONS=--loader=testdouble c8 tap --no-cov",
```

Comment:
both c8 and tap are not installed and there are no tests in this package, so I removed the script